packages/story-editor/src/app/media/utils/useDetectBaseColor.js

```diff
@@ -38,29 +38,23 @@ function useDetectBaseColor({ updateMediaElement }) {
     capabilities: { hasUploadMediaAction },
   } = useConfig();
   const { getProxiedUrl } = useCORSProxy();
-  const setProperties = useCallback(
```

Comment:
This same snippet is in a number places. If we remove here, should be removed everywhere!

Reply:
That's a nice to have.

I only changed it here because I was already touching this file.

Any other changes can be done in other PRs. P3 though.